bool approach_traj_started
bool approach_traj_finished

bool traj_started
bool traj_finished

bool send_pos
bool send_vel

float64[] pos_ref
float64[] vel_ref

uint16 performed_traj_n
